test/testdata/bootstrappolicy/bootstrap_cluster_role_bindings.yaml

```diff
@@ -305,7 +305,6 @@ items:
     apiGroup: rbac.authorization.k8s.io
     kind: ClusterRole
     name: system:node
-  subjects: null
 - apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
follow-up to get this in v1 format so it works with reconcile?

Reply:
> follow-up to get this in v1 format so it works with reconcile?

already in teh list